src/Coverage.Report.Tests/ConvertToLcovTaskTests.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Intentional to keep test code in `src/` directory? Would it not clutter structure when we start adding test assets?

Reply:
Yes, It is intentional. Copied from https://github.com/dotnet/corefx/tree/master/src/System.Buffers and some other great repos. TestAssets can have parallel to `src` dir?

Reply:
Just now one good thing I observed having tests in `test` directory is all the product code changes appears first while reviewing the code. Let's think about it. I'm open to both the options.